<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мир фишек</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

background-color: rgb(255, 241, 191);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Manrope</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'Manrope';
font-style: normal;

text-align: center;

background: rgba(140, 101, 65, 0.7);
border: 2px solid rgba(140, 101, 65, 0.7);
border-radius: 10px;
padding: 3px;</string>
     </property>
     <property name="text">
      <string>НАЗАД</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Icon1">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Money/pngegg (1) 1.png);
object-fit: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Logo">
       <property name="font">
        <font>
         <family>Manrope</family>
         <pointsize>45</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));</string>
       </property>
       <property name="text">
        <string>МИР  ФИШЕК</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Icon2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/Money/pngegg (1) 1.png);
background-fit: fit;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Bonus">
     <property name="font">
      <font>
       <family>Manrope</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">position: absolute;
width: 1061.33px;
height: 154.46px;
left: 105.19px;
top: 22.74px;

font-family: 'Manrope';
font-style: normal;
font-weight: 800;
font-size: 52.1191px;
line-height: 71px;
display: flex;
align-items: center;
text-align: center;

color: #462208;</string>
     </property>
     <property name="text">
      <string>ЗАРЕГИСТРИРУЙСЯ И ПОЛУЧИ БОНУС!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Email">
       <property name="font">
        <font>
         <family>Manrope</family>
         <pointsize>15</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">position: absolute;
width: 387.37px;
height: 75.64px;
left: 0px;
top: 13.75px;

font-family: 'Manrope';
font-style: normal;
font-weight: 800;
font-size: 34.382px;
line-height: 47px;
display: flex;
align-items: center;
text-align: center;

color: #462208;</string>
       </property>
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Edit_Email">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 216, 116);
border-radius: 15px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Password">
       <property name="font">
        <font>
         <family>Manrope</family>
         <pointsize>15</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">position: absolute;
width: 387.37px;
height: 75.64px;
left: 0px;
top: 13.75px;

font-family: 'Manrope';
font-style: normal;
font-weight: 800;
font-size: 34.382px;
line-height: 47px;
display: flex;
align-items: center;
text-align: center;

color: #462208;</string>
       </property>
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Edit_Password">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 216, 116);
border-radius: 15px</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="Sign_up">
     <property name="font">
      <font>
       <family>Manrope</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: 'Manrope';
font-style: normal;

text-align: center;

background: rgba(140, 101, 65, 0.7);
border: 2px solid rgba(140, 101, 65, 0.7);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>ЗАРЕГИСТРИРОВАТЬСЯ</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="Chips">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Money/money.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/project.qrc"/>
 </resources>
 <connections/>
</ui>
